content/en/docs/project-user-guide/custom-application-monitoring/get-started/monitor-mysql.md

```diff
@@ -0,0 +1,99 @@
+
+## Prerequisites
+
+- Please make sure you enable the OpenPitrix system. We will deploy MySQL and MySQL exporter from the App Store.
+- You need to create a workspace, a project, and a user account for this tutorial. The account needs to be a platform regular user and to be invited as the project operator. In this tutorial, we will work with the workspace my-workspace, and the project demo. We will log in as `project-operator`.
+
+## Hands-on Lab
+
+### Step 1: Deploy MySQL
+
+To begin with, we deploy MySQL from the App Store and set the root password to `testing`. Please make sure you are landing on the project overview page of `demo`.
+
+- Go to **App Store**
+
+![go-to-app-store](/images/docs/project-user-guide/custom-application-monitoring/go-to-app-store.PNG)
+
+- Find **MySQL** and Click **Deploy**
+
+![find-mysql](/images/docs/project-user-guide/custom-application-monitoring/find-mysql.PNG)
+
+![click-deploy](/images/docs/project-user-guide/custom-application-monitoring/click-deploy.PNG)
+
+- Make sure MySQL to be deployed in `demo`
+
+![click-next](/images/docs/project-user-guide/custom-application-monitoring/click-next.PNG)
+
+- Uncomment the `mysqlRootPassword` field and click **Deploy**
+
+![uncomment-mysqlRootPassword](/images/docs/project-user-guide/custom-application-monitoring/uncomment-mysqlRootPassword.PNG)
+
+- Wait until MySQL is up
+
+![check-if-mysql-is-running](/images/docs/project-user-guide/custom-application-monitoring/check-if-mysql-is-running.PNG)
+
+
+### Step 2: Deploy MySQL exporter
+
+Deploy MySQL exporter in `demo` on the same cluster. MySQL export is responsible for querying MySQL status and reports them in Prometheus format.
+
+- Go to **App Store** and find **MySQL exporter**
+
+![find-mysql-exporter](/images/docs/project-user-guide/custom-application-monitoring/find-mysql-exporter.PNG) 
+
+![exporter-click-deploy](/images/docs/project-user-guide/custom-application-monitoring/exporter-click-deploy.PNG)
+
+- Deploy MySQL exporter in `demo` again
+
+![exporter-click-next](/images/docs/project-user-guide/custom-application-monitoring/exporter-click-next.PNG)
+
+- Set `serviceMonitor.enabled` to true
+
+The builtin MySQL exporter sets it true by default, so you don't have to manually modify `serviceMonitor.enabled`.
+
+![set-servicemonitor-to-true](/images/docs/project-user-guide/custom-application-monitoring/set-servicemonitor-to-true.PNG)
+
+{{< notice warning >}}
+Don't forget to enable SericeMonitor CRD if you are using external exporter helm charts. Those charts usually disable ServiceMonitor by default and require manual modification.
+{{</ notice >}}
+
+- Modify MySQL connection parameters
+
+MySQL exporter needs to connect to the target MySQL. In this tutorial, MySQL is installed with the Service name `mysql-egn701`. So we modify `mysql.host` to `mysql-egn701`, `mysql.pass` to `testing`, and `user` to `root`. Note that your MySQL Service may be created with a different name!
```

Comment:
So we modify `mysql.host` to `mysql-egn701`, `mysql.pass` to `testing`, and `user` to `root`. Note that your MySQL Service may be created with a different name!
->
Modify `mysql.host` to `mysql-egn701`, `mysql.pass` to `testing`, and `user` to `root` as below. Note that your MySQL Service may be created with **a different name**.

Reply:
done